images/ose-agent-installer-api-server.yml

```diff
@@ -9,27 +9,25 @@ content:
     ci_alignment:
       streams_prs:
         ci_build_root:
-          stream: rhel-8-golang-1.20-ci-build-root
+          stream: rhel-9-golang-1.21-ci-build-root
 distgit:
-  branch: rhaos-{MAJOR}.{MINOR}-rhel-8
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
   component: ose-agent-installer-api-server-container
 enabled_repos:
-- rhel-8-baseos-rpms
-- rhel-8-appstream-rpms
-- rhel-8-server-ose-rpms-embargoed
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
+- rhel-9-server-ose-rpms-embargoed
+- rhel-9-codeready-builder-rpms
 for_payload: true
 from:
   builder:
-  - stream: golang-1.20
-  - member: openshift-enterprise-cli
-  member: openshift-enterprise-base
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
 maintainer:
   component: Installer
   subcomponent: Agent based installation
-name: openshift/ose-agent-installer-api-server
+name: openshift/ose-agent-installer-api-server-rhel9
 payload_name: agent-installer-api-server
 owners:
 - ocp-agent-team@redhat.com
-non_shipping_repos:
-- rhel-8-server-ose-rpms-embargoed
 canonical_builders_from_upstream: False # member image does not get resolved well
```

Comment:
This can now be removed, as we're no longer using a non -resolvable base image

Reply:
Right, updated 👍🏼